diff --git a/node_modules/react-native-vision-camera/ios/Core/CameraConfiguration.swift b/node_modules/react-native-vision-camera/ios/Core/CameraConfiguration.swift
index a4b94dd..a45af2b 100644
--- a/node_modules/react-native-vision-camera/ios/Core/CameraConfiguration.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/CameraConfiguration.swift
@@ -83,8 +83,7 @@ final class CameraConfiguration {
   /**
    Throw this to abort calls to configure { ... } and apply no changes.
    */
-  @frozen
-  enum AbortThrow: Error {
+  public enum AbortThrow: Error {
     case abort
   }
 
@@ -148,8 +147,7 @@ final class CameraConfiguration {
     }
   }
 
-  @frozen
-  enum OutputConfiguration<T: Equatable>: Equatable {
+  public enum OutputConfiguration<T: Equatable>: Equatable {
     case disabled
     case enabled(config: T)
 
diff --git a/node_modules/react-native-vision-camera/ios/Core/Recording/Track.swift b/node_modules/react-native-vision-camera/ios/Core/Recording/Track.swift
index 8dec5ce..b738483 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Recording/Track.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Recording/Track.swift
@@ -11,8 +11,7 @@ import Foundation
 
 // MARK: - TrackType
 
-@frozen
-enum TrackType {
+public enum TrackType {
   case audio
   case video
 }
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/AutoFocusSystem.swift b/node_modules/react-native-vision-camera/ios/Core/Types/AutoFocusSystem.swift
index b462c69..843cf56 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/AutoFocusSystem.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/AutoFocusSystem.swift
@@ -9,8 +9,7 @@
 import AVFoundation
 import Foundation
 
-@frozen
-enum AutoFocusSystem: String, JSUnionValue {
+public enum AutoFocusSystem: String, JSUnionValue {
   case contrastDetection = "contrast-detection"
   case phaseDetection = "phase-detection"
   case none
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/Flash.swift b/node_modules/react-native-vision-camera/ios/Core/Types/Flash.swift
index f0d97c2..951dd88 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/Flash.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/Flash.swift
@@ -11,8 +11,7 @@ import Foundation
 /**
  A Flash for Photo capture.
  */
-@frozen
-enum Flash: String, JSUnionValue {
+public enum Flash: String, JSUnionValue {
   /**
    Flash never fires.
    */
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/HardwareLevel.swift b/node_modules/react-native-vision-camera/ios/Core/Types/HardwareLevel.swift
index 98df469..644b162 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/HardwareLevel.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/HardwareLevel.swift
@@ -7,8 +7,7 @@
 
 import Foundation
 
-@frozen
-enum HardwareLevel: String, JSUnionValue {
+public enum HardwareLevel: String, JSUnionValue {
   case full
 
   init(jsValue: String) throws {
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/OutputOrientation.swift b/node_modules/react-native-vision-camera/ios/Core/Types/OutputOrientation.swift
index 4cb24fb..47c300a 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/OutputOrientation.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/OutputOrientation.swift
@@ -7,8 +7,7 @@
 
 import Foundation
 
-@frozen
-enum OutputOrientation: String, JSUnionValue {
+public enum OutputOrientation: String, JSUnionValue {
   /**
    Automatically rotate outputs based on device physical rotation (even if screen-lock is on)
    */
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/PixelFormat.swift b/node_modules/react-native-vision-camera/ios/Core/Types/PixelFormat.swift
index 83a43b9..9c0666c 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/PixelFormat.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/PixelFormat.swift
@@ -9,8 +9,7 @@
 import AVFoundation
 import Foundation
 
-@frozen
-enum PixelFormat: String, JSUnionValue {
+public enum PixelFormat: String, JSUnionValue {
   case yuv
   case rgb
   case unknown
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/QualityBalance.swift b/node_modules/react-native-vision-camera/ios/Core/Types/QualityBalance.swift
index 7db483a..cdd8818 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/QualityBalance.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/QualityBalance.swift
@@ -9,8 +9,7 @@
 import AVFoundation
 import Foundation
 
-@frozen
-enum QualityBalance: String, JSUnionValue {
+public enum QualityBalance: String, JSUnionValue {
   case speed
   case balanced
   case quality
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/ResizeMode.swift b/node_modules/react-native-vision-camera/ios/Core/Types/ResizeMode.swift
index 5317cab..aafcc74 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/ResizeMode.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/ResizeMode.swift
@@ -12,8 +12,7 @@ import Foundation
 /**
  A ResizeMode used for the PreviewView.
  */
-@frozen
-enum ResizeMode: String, JSUnionValue {
+public enum ResizeMode: String, JSUnionValue {
   /**
    Keep aspect ratio, but fill entire parent view (centered).
    */
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/ShutterType.swift b/node_modules/react-native-vision-camera/ios/Core/Types/ShutterType.swift
index 65121d1..a0dea4a 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/ShutterType.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/ShutterType.swift
@@ -11,8 +11,7 @@ import Foundation
 /**
  Represents the type of media that was captured in a `onShutter` event
  */
-@frozen
-enum ShutterType: String, JSUnionValue {
+public enum ShutterType: String, JSUnionValue {
   /**
    A photo was captured on this `onShutter` event
    */
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/Torch.swift b/node_modules/react-native-vision-camera/ios/Core/Types/Torch.swift
index 30c2be9..6b16ff6 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/Torch.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/Torch.swift
@@ -12,8 +12,7 @@ import Foundation
 /**
  A Torch used for permanent flash.
  */
-@frozen
-enum Torch: String, JSUnionValue {
+public enum Torch: String, JSUnionValue {
   /**
    Torch (flash unit) is always off.
    */
diff --git a/node_modules/react-native-vision-camera/ios/Core/Types/VideoStabilizationMode.swift b/node_modules/react-native-vision-camera/ios/Core/Types/VideoStabilizationMode.swift
index 9ff055e..aaa0483 100644
--- a/node_modules/react-native-vision-camera/ios/Core/Types/VideoStabilizationMode.swift
+++ b/node_modules/react-native-vision-camera/ios/Core/Types/VideoStabilizationMode.swift
@@ -9,8 +9,7 @@
 import AVFoundation
 import Foundation
 
-@frozen
-enum VideoStabilizationMode: String, JSUnionValue {
+public enum VideoStabilizationMode: String, JSUnionValue {
   case off
   case standard
   case cinematic
